### PR TITLE
Add entities size metric in querynode

### PR DESCRIPTION
**Pull request by bigsheeper**
/kind improvement

### CODE REVIEW

**Comment by bigsheeper** [comment]
rerun ut

File: pkg/metrics/querynode_metrics.go [hunk 1]
```diff
@@ -344,6 +344,18 @@ var (
 			indexCountLabelName,
 		})
 
+	QueryNodeSegmentSize = prometheus.NewGaugeVec(
```
Comment (czs007) [inline pkg/metrics/querynode_metrics.go:347]: Rename to QueryNodeEntitiesSize?

File: pkg/metrics/querynode_metrics.go [hunk 2]
```diff
@@ -344,6 +344,18 @@ var (
...
+		prometheus.GaugeOpts{
+			Namespace: milvusNamespace,
+			Subsystem: typeutil.QueryNodeRole,
+			Name:      "segment_size",
```
Comment (czs007) [inline pkg/metrics/querynode_metrics.go:351]: entity_size

File: pkg/metrics/querynode_metrics.go [hunk 3]
```diff
@@ -344,6 +344,18 @@ var (
...
+			Help:      "segment's memory size, clustered by collection and state",
```
Comment (czs007) [inline pkg/metrics/querynode_metrics.go:352]: entities' memory size, ...

**Pull request merged**